1.Exploratory_Data_Analysis/scripts/8.2_combine_files.py

```diff
@@ -45,20 +46,28 @@
 # get list of files in directory
 file_list = [x for x in data_dir.iterdir() if x.is_file()]
 
+list_of_dfs = []
 # loop through files
 for file in file_list:
     tmp_df = pd.read_parquet(file)
-    final_df = pd.concat([final_df, tmp_df])
-    # del the tmp_df to save memory
-    del tmp_df
-    # run garbage collection to free up memory
-    gc.collect()
+    list_of_dfs.append(tmp_df)
+final_df = pd.concat(list_of_dfs, ignore_index=True)
 
 final_df.shape
 
 
 # In[5]:
 
 
+# correct for multiple testing
+final_df["reject"], final_df["p-adj_fdr_bh"], _, _ = multipletests(
+    final_df["p-adj"], alpha=0.001, method="fdr_bh"
```

Comment:
I think the adjustment comes after performing `multipletests` consider renaming this to something like:
```suggestion
    final_df["p-unadj"], alpha=0.001, method="fdr_bh"
```

Reply:
I specify the multiple test_corrected column later on